test/Table_Tests/src/Database/Redshift_Spec.enso

```diff
@@ -4,7 +4,7 @@ import Standard.Base.Runtime.Ref.Ref
 import Standard.Table.Data.Type.Value_Type.Bits
 from Standard.Table import Table, Value_Type
 
-from Standard.Database import Database, SQL_Query
+from Standard.Database import all
```

Comment:
Why this change?

```suggestion
from Standard.Database import Database, SQL_Query, Update_Action
```
would have done?

Reply:
Other tests are importing `all` already, so I thought it can make them more uniform.

It's how we are importing stuff in the IDE, so I thought it makes the tests a bit 'closer to real life scenarios' this way.